src/packet.rs

```diff
@@ -192,7 +276,7 @@ impl MspParser {
 impl<'a> MspPacket<'a> {
 	/// Number of bytes that this packet requires to be packed
 	pub fn packet_size_bytes(&self) -> usize {
-		6 + self.data.len()
+		9 + self.data.len()
```

Comment:
This breaks MSP V1, if I understand it correctly. The packet size must be conditional based on whether it is V1 or V2.

Reply:
yes, you are correct, i think i will add different function packet_size_bytes_v2 for msp_v2

Reply:
You can also have a look here: https://github.com/wucke13/mavlink-msp-bridge/blob/856fa504e739ee440a7fa8d4986cdd6ea819d24f/src/msp.rs#L390

It's a encoder/decoder for MSPV2 which I wrote. Maybe you can copy some of it?
I would like to outsource that to it's own crate and combine my traits for encoding and decoding with the packed_struct stuff for all the MSP messages from this repo..

Reply:
yes, that sounds great.